http-api-data.cabal

```diff
@@ -25,25 +25,38 @@ library
     build-depends:   base               >= 4.6    && < 4.10
                    , text               >= 0.5
                    , bytestring
+                   , containers
+                   , mtl
                    , time
                    , time-locale-compat >=0.1.1.0 && <0.2
+                   , network-uri        >= 2.6    && < 2.7
+                   , string-conversions
```

Comment:
I don't think we need `string-conversions` for `http-api-data`. It used only for `encodeForm` and `decodeForm` and should be replaced with explicit conversion functions IMO.